core/src/main/java/com/datastrato/gravitino/listener/event/ExceptionEvent.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright 2024 Datastrato Pvt Ltd.
+ *  This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.listener.event;
+
+import com.datastrato.gravitino.annotation.Evolving;
+
+/** The base class for exception event. */
+@Evolving
+public abstract class ExceptionEvent extends Event {
```

Comment:
I think it is better to change to `FailureEvent`

Reply:
done